src/redux/categories/categories.js

```diff
@@ -0,0 +1,14 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] Going by the _duck patterns approach_, assigning value to _CHECK_STATUS_ action type should be something like:
```ruby
      CHECK_STATUS = bookstore/categories/CHECK_STATUS;
```
 
Kindly ensure to improve upon these aspects.

Reply:
Hi @rloterh, it's done, I forget that, thanks!!